var/spack/repos/builtin/packages/rocfft/package.py

```diff
@@ -34,6 +34,7 @@ class Rocfft(CMakePackage):
         depends_on('rocm-device-libs@' + ver, type='build', when='@' + ver)
         depends_on('hip@' + ver, when='@' + ver)
         depends_on('comgr@' + ver, type='build', when='@' + ver)
+        depends_on('comgr@' + ver, when='@' + ver)
```

Comment:
`comgr` is repeated?

Reply:
it requires link dependency too.. Modified accordingly